Library/Formula/agda.rb

```diff
@@ -51,38 +58,61 @@ def install
     cabal_clean_lib
 
     if build.with? "stdlib"
-      resource("stdlib").stage prefix/"agda-stdlib"
+      resource("stdlib").stage lib
```

Comment:
While some of the source files could possibly be deleted, there is a fine line between what is a useful, and what isn’t. I don’t want to introduce a brittle dependency on a particular version of the standard library, which is still evolving.

I thought `skip_clean "lib"` would prevent this complaint. I must be misunderstanding its purpose. Can you clarify?